cmd/arcstat/arcstat.py

```diff
@@ -112,7 +112,7 @@
 d = {}
 out = None
 kstat = None
-float_pobj = re.compile("^[0-9]+(\.[0-9]+)?$")
+float_pobj = re.compile(r'^[0-9]+(\.[0-9]+)?$')
```

Comment:
I think this is dead code and can be removed.

Reply:
Good catch, I've removed it.